code/EventLinkQR/app/src/main/java/com/example/eventlinkqr/AttendeesPage.java

```diff
@@ -65,7 +63,7 @@ public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceStat
         // make the back button return to the event page
         ((AppCompatActivity) requireActivity()).setSupportActionBar(orgAttendeesToolbar);
         orgAttendeesToolbar.setNavigationOnClickListener(v ->
-                Navigation.findNavController(view).navigate(R.id.action_attendeesPage_to_orgEventFragment));
+                Navigation.findNavController(view).navigate(R.id.action_orgAttendeesPage_to_orgEventFragment2));
```

Comment:
Why is does the id have "2" at the end?

Reply:
the ids got auto generated, i can change them to make them make more sense